lib/multiFetch.js

```diff
@@ -2,7 +2,7 @@ var request = require('superagent');
 var coBody = require('co-body');
 var batchRequestor = require('./batchRequestor')(require('./httpClient')(require('superagent')));
 
-module.exports = function (absolute) {
+module.exports = function (absolute, header_host) {
```

Comment:
Can you transform that into an options parameter and bump the version in package.json?